### PR TITLE
my first self review

### PR DESCRIPTION
**Pull request by NJ-2020**
A previous pretrial project to self review

### CODE REVIEW

**Comment by NJ-2020** [comment]
I think you may need to give a better space between syntaxes.

**Comment by NJ-2020** [comment]
Please use semantic HTML.

File: index.html [hunk 1]
```diff
@@ -14,18 +14,12 @@
...
-
-    - The freeCodeCamp Team
-
+    Microverse team
     -->
 
 
     <header>
-      <h1 id="title">Novel And Nadir Survey Form</h1>
+      <h1 id="title">Novel And Nadir Survey Form for own review</h1>
```
Comment (bolah2009) [inline index.html]: This is an example inline comment. 🙂 

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉 

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi @NJ-2020,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️


1. Let's use the GitHub inline comment. See the example inline comment review below.
2. I think your comments are not clear enough. Giving clear feedback helps with effective communication, instead of asking to make a change only also add "why" the changes are necessary. See this article on [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more details

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comment under one single summary like this current review. 😉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,76 +0,0 @@
...
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
-
```
Comment (NJ-2020) [inline index.html]: Please do not leave an unnecessary comment in your codebase as this may just confuse and can make things unclear.

**Review by NJ-2020 — COMMENTED** [review]
# Changes required ♻️ 

Please have a look at my comments below and make changes as required.

Resubmit when you are ready 😊

Regards

File: index.html (hunk 2 above)
Comment (acushlakoncept) [reply index.html]: Nice job on giving reasons why leaving unnecessary comments may not be a good idea.
However, you should pay attention more to students' commented code than generated comments. Example
```ruby
def func
  # student commented code
  # name = "Me"
  print "Me"
end
```

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
Hi @NJ-2020,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

You should always try to submit the PR link instead of the repo link.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Let's always start by following the "Request changes" template. I think this will help keep your request organized.

Linters passing is one of the fundamental requirements, try as much as possible to mention it in your review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Temmarie — APPROVED** [review]
# Status: Approved ✔️ 
## Hi @NJ-2020  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- Consider following the instructions given by the last reviewer and make use of Github Features when giving reviews.
- About the webhint linter errors, the purpose of this project is not to make it perfect, but to be able to point out errors and mistakes in your previous projects, suggesting ways in which it can be improved. Linters passing is one of the fundamental requirements, but the previous code reviewer only instructed you to mention it in your review.

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**